rocAL/rocAL_pybind/examples/rocAL_api_caffe_reader.py

```diff
@@ -35,7 +35,7 @@ def draw_patches(img, idx):
         image = img.cpu().numpy()
     else:
         image = img.detach().numpy()
-    image = image.transpose([1, 2, 0])
+    # image = image.transpose([1, 2, 0])
```

Comment:
Why is this line commented out?

Also, make sure to add the case for both NHWC & NCHW. 
Introduce arguments in draw_patches called dtype & layout to handle NHWC/ NCHW, fp32/ fp16/ uint8 cases.

Handle this in all the examples added

Reply:
@shobana-mcw Please advise on whether to keep the ported examples exactly same as ToT or add new functionalites for handling different layouts and dtypes